Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:20.04
+ENV DEBIAN_FRONTEND=noninteractive
+WORKDIR /sqlite
+RUN apt-get update && apt-get install -y git
+RUN git clone https://github.com/sqlite/sqlite.git .
+RUN git checkout version-3.45.1
+RUN apt-get install -y \
+autoconf \
+automake \
+bison \
+libtool \
+make \
+tcl \
+pkg-config \
+gcc \
+g++ \
+tcl-dev \
+zlib1g-dev \
+tcl8.6-dev \
+tar
+RUN cd /sqlite \
+&& git fetch \
+&& git checkout version-3.45.1 \
+&& ./configure \
+&& make \
+&& make sqlite3.c \
+&& make devtest
+EXPOSE 8080
```

Comment:
Seems like a sane default, but should this port be configurable?

Reply:
This port is a configurable value. It can be specified by the user before the tool starts running. If the user does not specify it, the tool looks for a possible port in the project file, and if it does not find one, it uses the default port. After the Dockerfile is generated, you can also modify the port number according to your needs.

Reply:
I would argue that the configurability of the port is worth documenting.